src/bin/daemon.rs

```diff
@@ -19,6 +19,10 @@ fn parse_args(args: Vec<String>) -> Option<PathBuf> {
         return None;
     }
 
+    if args[1] != "--conf" {
+        eprintln!("lianad take only one (mandatory) argument: --conf");
+    }
```

Comment:
No, it's not mandatory. :)

```suggestion
    if args[1] != "--conf" {
        eprintln!("Only a single command line argument is supported: --conf. All other configuration parameters must be specified in the configuration file.");
        print_help_exit();
    }
```